### ISSUE TITLE
Import/export fails

### ISSUE THREAD

**Issue by peterb776** [Desertsnowman/Caldera-Forms#67]
I'm using the latest version (1.1.9.3) on my development system (UniServer on laptop) and on the live system (Linux). I exported a form from the dev system and then imported it to the live system. There were no error messages but what I got was all the rows of the form, but none of the fields.

I have no way to determine whether the fault is in the export or import routine. Help please.

Thanks,
Peter

**Comment by Desertsnowman** [comment]
Can you send me the export file to david@caldera.co.za

**Comment by ahancey** [comment]
I have been having the same issue.  It is not with wordpress, it is with Caldera  Forms.  When I import a form that I have exported, all of the if-statements that I had created are missing.  I have gone through and fixed all of the if statement for each element and it works again, until I make a change at the top of the form.  If I add a new element at the top of the form, all of the if-statements below that are missing or incorrect.  I've tried to make this work for 2 weeks, but uninstalling now.  Too many bugs.
